app/code/Magento/LoginAsCustomerGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,30 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Mutation {
+    generateCustomerTokenAsAdmin(
+        input: GenerateCustomerTokenAsAdminInput!
+    ): GenerateCustomerTokenAsAdminOutput
+    @resolver(class: "Magento\\LoginAsCustomerGraphQl\\Model\\Resolver\\RequestCustomerToken")
+    @doc(description: "Request a customer token so that an administrator can perform remote shopping assistance")
+}
+
+input GenerateCustomerTokenAsAdminInput {
+    customer_email: String! @doc(description: "The email address of the customer requesting remote shopping assistance")
+}
+
+type GenerateCustomerTokenAsAdminOutput {
+    customer_token: String! @doc(description: "The generated customer token")
+}
+
+type Customer {
+    allow_remote_shopping_assistance: Boolean! @doc(description: "Indicates whether the customer has enabled remote shopping assistance")
```

Comment:
The GetCustomer resolver will need to be updated to return this new value

Reply:
Done

Reply:
Added a new fixture for this with setAssistanceAllowed enabled